psake.psm1

```diff
@@ -621,7 +621,23 @@ function ConfigureBuildEnvironment {
     $frameworkDirs = @()
     if ($buildToolsVersions -ne $null) {
         foreach($ver in $buildToolsVersions) {
-            if (Test-Path "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\$ver") {
+            if ($ver -eq "15.0") {
+                $vs2017paths = @(
+                    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise",

```

Comment:
Please note that Enterprise, Professional, and Community is not enough.
Another known installation is BuildTools.
See https://github.com/nightroman/Invoke-Build/pull/57